src/store/state/state-model.ts

```diff
@@ -92,7 +92,7 @@ export const stateModel: StateModelIF = {
   documentDelivery: {
     documentOptionalEmail: ''
   },
-  officeAddresses: {},
+  officeAddresses: null,
```

Comment:
The old value supported de-referencing but null doesn't. As long as this is always initialized somehow, this is fine (and preferable). 👍